models/risk_models.py

```diff
@@ -125,7 +139,62 @@ def update_risk_encounter(cls, human, message):
         human.M[msg_enc]['carry_over_transmission_proba'] = RISK_TRANSMISSION_PROBA * (1 - update)
 
 
+class RiskModelTristan(RiskModelBase):
+    risk_map = np.load("models/log_risk_mapping.npy")
```

Comment:
`risk_map = np.load(f"{os.path.dirname(os.path.realpath(__file__))}/log_risk_mapping.npy")`

not sure if it is cleaner but at least it allows to start the preproc in a unittest runner which has its own working dir

Reply:
fine by me, we could also (definitely) just store this in an array in the code.